openrr-client/src/error.rs

```diff
@@ -5,22 +5,26 @@ use urdf_rs::UrdfError;
 #[derive(Debug, Error)]
 #[non_exhaustive]
 pub enum Error {
-    #[error("openrr-cleint: No JointTrajectoryClient={} is found.", .0)]
-    NoJointTrajectoryClient(String),
+    #[error("openrr-cleint: arci: {:?}", .0)]
```

Comment:
Do you know why this soring happed?

Reply:
I did it manually, since I cannot find any orders to define `Errors`. 

If you have a problem, I'll revert it.

Reply:
If it is your intention (not a mistake, merge conflict nor something), it's OK.